### ISSUE TITLE
Streaming Catalogs

### ISSUE THREAD

**Issue by rleroi** [danamag/stremio-addons-list#87]
### Addon Manifest URL

https://7a82163c306e-stremio-netflix-catalog-addon.baby-beamup.club/manifest.json

### Addon Description

Trending movies and series on Netflix, HBO Max, Disney+, Hulu and more. Configure to choose your favourite services.

### Language of Content

Multilingual (Default)

### Choose Labels

- [X] anime
- [X] asian drama
- [X] bollywood
- [X] cartoons
- [ ] debrid support
- [ ] http streams
- [ ] live tv
- [X] metadata
- [ ] misc
- [X] movies
- [X] music
- [ ] nsfw
- [ ] podcasts
- [ ] radios
- [ ] subtitles
- [ ] torrents
- [X] tv shows
- [ ] video games

**Comment by rleroi** [comment]
Go to https://7a82163c306e-stremio-netflix-catalog-addon.baby-beamup.club/ to configure what streaming services you want to see.

<img width="320" alt="image" src="https://user-images.githubusercontent.com/6817390/205947778-d1247e61-cffc-43c0-916a-9000f91ecaed.png">

**Comment by rleroi** [comment]
#Update v1.0.5

- Updated providers and filtering of providers by country
- Added Rating Posters Database (RPDB) option

<img width="324" alt="image" src="https://user-images.githubusercontent.com/6817390/222761620-ed20d983-d32d-48f6-be34-bab5ac9c4233.png">

**Comment by PadreToio** [comment]
Amazing add-on, love it.

Not sure if it's something on your side or Stremio... Is it possible to add the "Title (Year)" below the Poster when browsing content?

**Comment by rleroi** [comment]
Canal+ and Discovery+ added

**Comment by DryKillLogic** [comment]
The addon is not currently working. Please use the patched fork until it gets fixed.

https://848b3516657c-stremio-streaming-catalogs-reborn-addon.baby-beamup.club/

**Comment by rleroi** [comment]
Should be working again

**Comment by MrSick0ff** [comment]
This one is amazing, but I wish we had option to pick whether we want to see shows, movies or both from each category.

**Comment by a95tvbox** [comment]
Hey Bro @rleroi , I'm having an issue which is...the thumbnails are not showing...can you help me with this...this addon is one of the best...please help me to fix this
![issue 2](https://github.com/danamag/stremio-addons-list/assets/158166844/919d0b91-8a3d-4c2c-bcbe-c903c70c2406)

**Comment by a95tvbox** [comment]
> ### Addon Manifest URL
> https://7a82163c306e-stremio-netflix-catalog-addon.baby-beamup.club/manifest.json
> 
> ### Addon Description
> Trending movies and series on Netflix, HBO Max, Disney+, Hulu and more. Configure to choose your favourite services.
> 
> ### Language of Content
> Multilingual (Default)
> 
> ### Choose Labels
> * [x]  anime
> * [x]  asian drama
> * [x]  bollywood
> * [x]  cartoons
> * [ ]  debrid support
> * [ ]  http streams
> * [ ]  live tv
> * [x]  metadata
> * [x]  misc
> * [x]  movies
> * [ ]  music
> * [ ]  nsfw
> * [ ]  podcasts
> * [ ]  radios
> * [ ]  subtitles
> * [ ]  torrents
> * [x]  tv shows
> * [ ]  video games

![issue 2](https://github.com/danamag/stremio-addons-list/assets/158166844/8df8798d-67da-48c7-9a1a-5eea8773dceb)
Help me bro @rleroi

**Comment by vijayvcm** [comment]
Filter by language option?

In India, There are many languages. Choosing just India from dropdown doesn’t helpful. We need another dropdown to filter out language specific contents.

**Comment by Raelag83** [comment]
Hi , I have recently isntalled the addon. I cans ee only 99 movie when selecting hotstar. It seems its loading part of the catalog. I have tried installing /configuring several time but nothing has changed. Can someone please confirm if it doesn't load the  catalog

**Comment by MrsKryten** [comment]
RPDB now provides keys to free users. I am trying to enter my free RPDB key to the Streaming Catalogs configuration page, but it refuses it with a "Please match the requested format" message. The free key works on other Stremio add-ons though. I'm not sure if the format is different from a RPDB subscription-based key and there is some filter on SC's website that is blocking the free ones.

**Comment by sertelovic** [comment]
Can anyone solve the free RPDB problem?

**Comment by rleroi** [comment]
Free RPDB should now work.
Also, Funimation has been removed and Crunchyroll has been added back.

**Comment by MrsKryten** [comment]
> Free RPDB should now work. Also, Funimation has been removed and Crunchyroll has been added back.

Thank you for the fix! I appreciate it.

**Comment by Dude138** [comment]
Nflix only showing kids shows and movies now.

**Comment by sertelovic** [comment]
Thank you for the fix.. but only nflix showing kids show and movies. I am sure you will fix it soon. thanks in advance .

**Comment by rleroi** [comment]
@Dude138 @sertelovic Yep I've accidentally replaced Netflix with Netflix Kids. It's fixed and we now have Netflix Kids added in the configuration page :)
![image](https://github.com/user-attachments/assets/8dde3c18-fe64-4aae-8e94-282a7a490a02)

**Comment by sertelovic** [comment]
@rleroi  thank you so much for fix. And thank you so much one of the best add on for stremio :)

**Comment by razeNFR** [comment]
Hello, very good addon i knew it today and i prefer it to cyberflix, but, you think its possible to add language selection for synopsis of movie/serie too?

**Comment by rleroi** [comment]
@razeNFR Try it, it's open source :) But it would require a bit of a rewrite. https://github.com/rleroi/Stremio-Streaming-Catalogs-Addon

**Comment by Djlilyazii** [comment]
crunchyroll is not updating anymore please fix it
